docs/source/pages/Simulator.rst

```diff
@@ -65,5 +65,3 @@ Real trader
 -----------
 
 Additionally to the simulated trading system, a real trader is available in OctoBot.
-
-`Here is the article describing the real trader feature of OctoBot <Trader.html>`_
```

Comment:
why deleting it ? my previous question vas about this change (Trader.html to Profile-Configuration.html in link):
![image](https://user-images.githubusercontent.com/9078616/119269897-4f34a380-bbfa-11eb-8bf4-8ca42a81d9ad.png)

Reply:
Because Trader configuration has been moved in Profile > Trading